### DISCUSSION TITLE
Delete beatmaps that exists on stable

### DISCUSSION THREAD

**Discussion by FriendsNone** [ppy/osu#21964] [Ideas] [answered]
Now that hard links are in latest Windows builds of lazer. I want to take advantage of the feature, as I actively play on both versions.

I don't know what's the current behavior of importing beatmaps from stable. But I'd assume it only works on new installs and/or new beatmaps on stable that hasn't been imported yet in latest lazer.

If assumptions were correct. I'd suggest that:
- importing from stable checks if the beatmap(s) is already in lazer
- then replace it with a hard link version of the beatmap(s)

If it isn't, then I'd suggest:
- implementing a delete all beatmaps button, but it only deletes the ones that only exists on stable...
- or deletes the ones that aren't downloaded/imported through lazer

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21827
